pkg/gui/commits_panel.go

```diff
@@ -462,21 +462,30 @@ func (gui *Gui) handleCommitPick() error {
 }
 
 func (gui *Gui) handleCommitRevert() error {
-	if ok, err := gui.validateNotInFilterMode(); err != nil || !ok {
-		return err
-	}
-
 	commit := gui.getSelectedLocalCommit()
+	return gui.ask(askOpts{
+		title: gui.Tr.Actions.RevertCommit,
+		prompt: utils.ResolvePlaceholderString(
+			gui.Tr.ConfirmRevertCommit,
+			map[string]string{
+				"selectedCommit": commit.ShortSha(),
+			}),
+		handleConfirm: func() error {
+			if ok, err := gui.validateNotInFilterMode(); err != nil || !ok {
```

Comment:
we're better off keeping this function call where it was before. The purpose of that check is to disable a bunch of keybindings when we're in filter mode